static/src/components/task_browse/assign_worker.vue

```diff
@@ -103,8 +102,24 @@ export default {
     },
 
     async save () {
+      let requestData = {
+        taskId: this.getSelectedTask(),
+        filters: JSON.stringify(this.getFilters()),
+        add: this.addUserValues,
+        remove: this.removeUserValues
+      };
       try {
         this.waiting = true;
+        const res = await fetch(this.getURL(), {
+          method: 'POST',
+          headers: {
+            'content-type': 'application/json',
+            'X-CSRFToken': this.getCsrfToken()
+          },
+          credentials: 'same-origin',
+          body: JSON.stringify(requestData)
+        });
+        console.log(res);
```

Comment:
Just confirming if we want to include the `console.log(res)` (for debugging?) or if it should be removed.

Reply:
I believe we can remove the console.log(res) as well as the assignment to res. Thanks!

Reply:
In other components, we validate that `res` is not an error response. For example, 
https://github.com/bloomberg/pybossa-default-theme/blob/787afda1e2149d284763a1e866e35416e281f8dd/static/src/components/setting/ownership_setting.vue#L206-L212

Reply:
One suggestion is to keep the try/catch as it is. Additionally, we could replace the `console.log()` with the code excerpt above for checking the `res.ok` value. This would resolve the travis issue as well as give a more detailed error message to the user.